skimage/data/__init__.py

```diff
@@ -13,7 +13,9 @@
                       'immunohistochemistry', 'kidney',
                       'lbp_frontal_face_cascade_filename', 'lily',
                       'lfw_subset', 'logo', 'microaneurysms', 'moon', 'page',
+                      'protein_transport',
                       'text', 'retina', 'rocket', 'shepp_logan_phantom',
-                      'skin', 'stereo_motorcycle', 'vortex']
+                      'skin', 'nickel_solidification', 'stereo_motorcycle',
```

Comment:
not too big of a deal, but it looks like these were mostly alphabetical previously? (although `text` was out of place).

Reply:
Oh, sorry, that's because I first named it 'solidification' :sweat_smile: ('lily' was also out of place). Reordering the entire list... (except for 'data_dir', 'download_all')